install-sbo.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+set -e
+set -u
+REGISTRY_PASSWORD?=""
+
+#The SBO can also be installed in a vanilla kubernetes cluster. A prerequisite for this would be to add credentials for the registry.redhat.io in the cluster. Steps to be followed to achieve the same are listed below:
+#1) Follow [Red Hat Container Registry Authentication Steps](https://access.redhat.com/RegistryAuthentication)
+#2) Verify that your credentials are correct using docker login -u <your_username> -p <your_passwd> registry.redhat.io
+echo "${REGISTRY_PASSWORD}" | docker login -u "$USERNAME" --password-stdin registry.redhat.io
+
+#3) Start a kubernetes cluster. For this case we have used minikube. Any other Kubernetes cluster can be used as well.
+minikube start
```

Comment:
The script should assume that a cluster is up and running - we should not spin up cluster here. This would enable us to use the same script against various clusters.

Reply:
Done